fix(ocpp): avoid an error log in handling downstream messages (#12892)
emqx/emqx: changes/ee/fix-12892.md

> Fix a logical error in OCPP gateway's handling of downstream BootNotification.

Reviewer: ```suggestion
Fix an error in OCPP gateway's handling of downstream BootNotification.
```

> Fix the `gateways/ocpp/listeners` endpoint to return the correct current connection number.

Reviewer: ```suggestion
Fix the `gateways/ocpp/listeners` endpoint to return the correct number of current connections.
```
